web-app/src/components/index.js

```diff
@@ -22,3 +22,6 @@ export { default as HolidayModal } from './HolidayModal';
 export { default as AllEmployees } from './AllEmployees';
 export { default as DataTable } from './DataTable';
 export { default as FullPageLoader } from './FullPageLoader';
+export { default as TeamOverview } from './TeamOverview';
+
+
```

Comment:
Small comment! But could you remove the extra line here, apologies to be pedantic :)